config/app.php

```diff
@@ -312,8 +307,6 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
-
-            'url' => env('DATABASE_URL', null),
```

Comment:
`env('DATABASE_URL', null)` should leave in `app.php`, so it would work with environment variables, but no `app_local.php`.

Reply:
This point of this change is to not have hidden db credentials in app.php.  Users should add 'url' if they want DATABASE_URL to permanently override.

Reply:
Having it stubbed in the local file sounds good.

Reply:
While I believe (actually, I hope) that I understand the reason for having app_local.php, but wouldn't it make more sense to push (and promote) using .env file for local development?

Understanding that this might mean that more developers be educated about using environment variables in non-familiar hosting setups, but certain clear ideas can definitely be floated how to use .env in local setups while using same code base (no ignoring any other files, apart from .env file) and have a same-code setup on all environments (Local, Dev, QA, UAT, Prod).

Reply:
Not everyone can / want to use environment variables in production. For e.g. apps deployed on shared hosting.